package.json

```diff
@@ -3,17 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@babel/polyfill": "^7.2.5",
```

Comment:
@jaworek, I just wanted to double-check before you merge as this could cause issues with IE11. If we don't support that browser, then we should be alright to remove this polyfill.

Reply:
This application uses create-react-app. I think it because it hides its babel configuration it will not work. I found this wiki page that add all needed polyfills for older browsers.
https://github.com/facebook/create-react-app/blob/master/packages/react-app-polyfill/README.md

https://create-react-app.dev/docs/supported-browsers-features/

Reply:
I can add it if support for IE11 is necessary.

Reply:
https://babeljs.io/docs/en/babel-polyfill
This package is also deprecated. I think it is better to remove or replace it.